src/api/spec/controllers/webui/main_controller_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+RSpec.describe Webui::MainController do
+  let(:user) { create(:confirmed_user) }
```

Comment:
When you create a user a home project got created at the same time... so the name of the home project is based on the user login, supposedly there is a call to the backend trying to create a project with a "random" name... right?